Source/Fuse.Controls.CameraView/CameraViewBase.uno

```diff
@@ -33,6 +33,10 @@ namespace Fuse.Controls
 		Front,
 	}
 
+	public enum CameraFocusPoint {
+		Ok,
+	}
```

Comment:
An enum with a single value seems nonsensical to me... What's the point of this?

Reply:
Looking over the code again, I didn't understand how to write in uno that the value returned for a function could be an int if successful or a string if error, so I think that's why I created the enum to promise "something"